setup.py

```diff
@@ -296,4 +293,5 @@ def run(self):
         distclass=Distribution,
         cmdclass=cmdclass,
         python_requires='>=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
+        install_requires=['future',],
```

Comment:
Instead of using future, what do you think about using [six](https://github.com/benjaminp/six)? From my experience, it is much more commonly used and better maintained library by a  core Python developer. 

Using [pypinfo](https://github.com/ofek/pypinfo), we can show the download statistic from the last 30 days. They show six to be the much more popular choice for a Python 2/3 compatibility layer.

| project | download_count |
| ------- | -------------: |
| six     |     35,049,126 |
| future  |      6,616,141 |

Reply:
plus python-future can have some nasty side-effects on other libraries: [1](https://github.com/pytest-dev/pytest/pull/4074) [2](https://github.com/Yelp/yelp_clog/pull/18#issue-64272423) and is a much larger footprint

Reply:
@jdufresne I'm ok with using six; I forget whey I passed over it in the first
place but it was on my radar.

Reply:
+1 for six